dissect/target/helpers/record.py

```diff
@@ -175,7 +175,7 @@ def DynamicDescriptor(types):  # noqa
     [
         *COMMON_INTERFACE_ELEMENTS,
         ("varint", "vlan"),
-        ("string", "proxy"),
+        ("net.ipaddress[]", "proxy"),
```

Comment:
As I don't think you'll want to add it in this PR, can you add a ticket to extend the macOS network interface plugin with proxy records that include the type, port and exclusions.

Reply:
DIS-3418

Reply:
Can you make it public too

Reply:
Set to public

Reply:
https://github.com/fox-it/dissect.target/issues/863